.yamato/_run-all.yml

```diff
@@ -35,7 +37,10 @@ all_package_tests:
 {% for editor in test_editors -%}
 {% for platform in test_platforms -%}
 {% for project in projects -%}
-    - .yamato/package-tests.yml#test_{{ projects.first.packages.first.name }}_{{ editor }}_{{ platform.name }}
+{% for package in project.packages -%}
+    - .yamato/package-tests.yml#test_{{ package.name }}_{{ editor }}_{{ platform.name }}
+{% endfor -%}
+    - .yamato/package-tests.yml#test_{{ projects.first.packages.second.name }}_{{ editor }}_{{ platform.name }}
```

Comment:
This is redundant with the loop on 41, is it not?

Reply:
Fixed